mappings/net/minecraft/client/gui/screen/narration/ToStringTransformer.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/class_6384 net/minecraft/client/gui/screen/narration/ToStringTransformer
+	FIELD field_33793 UNIT_TRANSFORMER Lnet/minecraft/class_6384;
```

Comment:
This isn't a transformer, but data packed with transformation logic, so I would just name this to indicate this is a data class, like `Narration` (not idea either, as this is essentially supplying a stream of sentences to a string consumer)